import-export-cli/cmd/exportAPI_test.go

```diff
@@ -19,16 +19,18 @@
 package cmd
 
 import (
-	"github.com/go-resty/resty"
+	"github.com/stretchr/testify/assert"
 	"os"
 	"path/filepath"
 	"testing"
 
-	"fmt"
-	"github.com/renstrom/dedent"
-	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"github.com/go-resty/resty"
+
 	"net/http"
 	"net/http/httptest"
+
```

Comment:
we need to check the license of these libraries

Reply:
`Resty` and `Testify` are licensed under MIT